akka-docs/rst/scala/code/docs/stream/ActorSubscriberDocSpec.scala

```diff
@@ -79,11 +82,13 @@ class ActorSubscriberDocSpec extends AkkaSpec {
 
     //#actor-subscriber-usage
     val N = 117
-    Source(1 to N).map(WorkerPool.Msg(_, replyTo))
+    val worker = Source(1 to N).map(WorkerPool.Msg(_, replyTo))
       .runWith(Sink.actorSubscriber(WorkerPool.props))
     //#actor-subscriber-usage
 
+    watch(worker)
     receiveN(N).toSet should be((1 to N).map(WorkerPool.Done).toSet)
```

Comment:
Thanks for reminding me. I have seen it and remember making changes. Changes are unpushed for some reason. I will get back to this right now.